### PR TITLE
Test data for OVS

### PR DESCRIPTION
**Pull request by kagemand-andersen**

### CODE REVIEW

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 1]
```diff
@@ -0,0 +1,116 @@
+\set ON_ERROR_STOP true
+\connect dcsa_openapi
+
+BEGIN;
+
+INSERT INTO dcsa_im_v3_0.port_call_service_type (
+    port_call_service_type_code,
+    port_call_service_type_name,
```
Comment (Abdiiir) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:8]: Does not respect defined maximum length varchar(30).
This applies to all `port_call_service_type_name` attributes. 
Error thrown -> `ERROR:  value too long for type character varying(30)`

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    'PILO_port_call_service_type_description'
+);
+
+INSERT INTO dcsa_im_v3_0.port_call_service_type (
+    port_call_service_type_code,
+    port_call_service_type_name,
+    port_call_service_type_description
+) VALUES (
+    'MOOR',
+    'MOOR_port_call_service_type_name',
```
Comment (Abdiiir) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:22]: same here and all the way down. 
Besides that it **LGTM**

**Review by Abdiiir — CHANGES_REQUESTED** [review]

File: datamodel/testdata.d/08_03_test_data_ovs.sql (hunk 1 above)
Comment (kagemand-andersen) [reply datamodel/testdata.d/08_03_test_data_ovs.sql:8]: Ah, I think this was changed in a commit in master that was merged afterwards. They have all been shortened.

**Review by kagemand-andersen — COMMENTED** [review]
Done

**Review by kagemand-andersen — COMMENTED** [review]
done?

**Comment by HenrikHL** [comment]
Remember to update the submodules in OVS in order to see if all tests succeed...

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+);
+
+INSERT INTO dcsa_im_v3_0.port_call_service_type (
+    port_call_service_type_code,
+    port_call_service_type_name,
+    port_call_service_type_description
+) VALUES (
+    'MOOR',
+    'MOOR_name',
+    'MOOR_description'
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:23]:
```suggestion
    'A better description for MOOR'
```

**Review by douradofrazer — DISMISSED** [review]

**Review by Abdiiir — APPROVED** [review]
Looks good

**Pull request merged**